### DISCUSSION TITLE
port conflict issue due to rabbitmq dynamic ports allocation

### DISCUSSION THREAD

**Discussion by satvindergill14** [rabbitmq/rabbitmq-server#11629] [Questions]
Hi Everyone,

we are facing port conflict issue while starting any docker container on Host as RabbitMQ is assigning below dynamic ports for internal communication , can you please help to get rid of this issue.

tcp        0      0 ::ffff:ContainerIP:5672 ::ffff:172.19.1.1:56091 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:56082 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:55963 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:56094 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:56095 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:56086 ESTABLISHED -
tcp        0      0 ::ffff: ContainerIP :5672 ::ffff:172.19.1.1:56077 ESTABLISHED -

can anyone help here , i want ports should not be dynamically allocated , how we can control the ports which RabbitMQ is using internally

**Comment by satvindergill14** [comment]
Hi Everyone , can you please help here

**Reply by mkuratczyk** [reply]
You've already received a reply on the mailing list (in the future, please don't open the same discussion in multiple places at the same time)
https://groups.google.com/g/rabbitmq-users/c/Kondt7mZeb4/m/tOdh0eorAgAJ
